clean.sh

```diff
@@ -46,28 +46,48 @@ fi
 		chmod +w -R "$TERMUX_TOPDIR" || true
 	fi
 
-	# For on-device build cleanup /data shouldn't be erased.
+	# For on-device build cleanup Termux app data directory shouldn't be erased.
 	if [[ "$TERMUX_ON_DEVICE_BUILD" == "false" ]]; then
-		if [[ ! "$TERMUX_BASE_DIR" =~ ^(/[^/]+)+$ ]]; then
-			echo "TERMUX_BASE_DIR '$TERMUX_BASE_DIR' is not an absolute path under rootfs '/'." 1>&2
-			exit 1
-		fi
-		if [[ "$TERMUX_BASE_DIR" =~ ^/[^/]+$ ]]; then
-			# Use `/rootfs` as is.
-			rootfs_top_level_dir="$TERMUX_BASE_DIR"
+		for variable_name in TERMUX__PREFIX TERMUX_APP__DATA_DIR CGCT_DIR; do
+			variable_value="${!variable_name:-}"
+			if [[ ! "$variable_value" =~ ^(/[^/]+)+$ ]]; then
+				echo "The $variable_name '$variable_value' is not an absolute path under rootfs '/' while running 'clean.sh'." 1>&2
```

Comment:
I feel like these two could use a rename.
And probably also a comment because I just had to reference the man page to remember that `${!var}` is an indirection, not a key lookup (`${!arr[i]}`).

Reply:
Which two and comment what? `${!var}` is pretty common and loop items are clearly variable names anyways being expanded to a value.